packages/next/src/shared/lib/turbopack/utils.ts

```diff
@@ -30,6 +33,18 @@ export class ModuleBuildError extends Error {
 export class TurbopackInternalError extends Error {
   name = 'TurbopackInternalError'
 
+  static createAndRecordTelemetry(cause: Error) {
+    const error = new TurbopackInternalError(cause)
+
+    const telemetry = traceGlobals.get('telemetry')
+    if (!telemetry) {
+      throw new Error('Expected `telemetry` to be set in globals')
```

Comment:
This would result in the `TurbopackInternalError` being swallowed. Maybe use `console.error` instead?